Rakefile

```diff
@@ -6,14 +6,19 @@ require "rake/testtask"
 require 'psych'
 
 desc "Setup Rubygems dev environment"
-task :setup => ["bundler:checkout"] do
+task :setup => ["git_hooks", "bundler:checkout"] do
```

Comment:
IMO, i really don't like it when projects automatically configure development workflows, it just ends up getting in the way most of time. Instead, i suggest documenting this git-hook that anyone can opt into if they wished.

Reply:
Fair 👍